androidgen/src/main/java/com/azure/autorest/android/model/clientmodel/AndroidClientMethod.java

```diff
@@ -64,6 +65,7 @@ public void addImportsTo(Set<String> imports, boolean includeImplementationImpor
         imports.add("com.azure.android.core.rest.annotation.ServiceMethod");
         imports.add("com.azure.android.core.rest.annotation.ReturnType");
         imports.add("java.util.concurrent.ExecutionException");
+        imports.add("java9.util.function.Function");
```

Comment:
Any specific reason to use java9 here?

Reply:
My understanding is we are targeting Android SDK 15 if possible.